src/http-handler.js

```diff
@@ -0,0 +1,190 @@
+import debugFactory from 'debug';
+import getRawBody from 'raw-body';
+import crypto from 'crypto';
+import { packageIdentifier } from './util';
+
+export const errorCodes = {
+  NO_BODY_PARSER: 'SLACKEVENTMIDDLEWARE_NO_BODY_PARSER',
+  SIGNATURE_VERIFICATION_FAILURE: 'SLACKMESSAGEMIDDLEWARE_REQUEST_SIGNATURE_VERIFICATION_FAILURE',
+  REQUEST_TIME_FAILURE: 'SLACKHTTPHANDLER_REQUEST_TIMELIMIT_FAILURE',
+};
+
+const responseStatuses = {
+  OK: 200,
+  FAILURE: 500,
+  REDIRECT: 302,
+  NOT_FOUND: 404,
+};
+
+const debug = debugFactory('@slack/events-api:http-handler');
+
+export function createHTTPHandler(adapter) {
+  // Removed middlewareOptions because no next() -- is this okay?
```

Comment:
i think yes, because `propogateErrors` is no longer an option, so it doesn't have a purpose. can be introduced later in a backwards-compatible way.